src/test/java/com/google/api/codegen/gapic/testdata/ruby/ruby_library.baseline

```diff
@@ -4970,311 +4970,888 @@ Gem::Specification.new do |gem|
-//FIXME: license section
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 
 # DO NOT EDIT! This is a generated sample ("Request",  "empty_response_type_without_response_handling")
-# FIXME: import section
-# FIXME: sample function declaration
-# FIXME: initialize client
 
-formatted_name = Library::V1::LibraryServiceClient.shelf_path("[SHELF_ID]")
+require "library"
+
+# [START sample]
+
+# Test default response handling is turned off for methods that return empty
+def sample_delete_shelf
+  # [START sample_core]
+  # Instantiate a client
+  library_service_client = Library.new version: :v1
+
+  formatted_name = library_service_client.class.shelf_path("[SHELF_ID]")
 
-library_service_client.delete_shelf(formatted_name)
-# FIXME: response handling
-# FIXME: main function
+  library_service_client.delete_shelf(formatted_name)
+
+  # [END sample_core]
+end
+# [END sample]
+
+
+require "optparse"
+
+if $0 == __FILE__
+  sample_delete_shelf
+end
 ============== file: samples/discuss_book_request_streaming_bidi_prog.rb ==============
-//FIXME: license section
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 
 # DO NOT EDIT! This is a generated sample ("RequestStreamingBidi",  "prog")
-# FIXME: import section
-# FIXME: sample function declaration
-# FIXME: initialize client
-
-name = "BASIC"
-file_name = "comment_file"
-comment = File.binread file_name
-comment_2 = { comment: comment }
-image = File.binread image_file_name
-request = {
-  name: name,
-  comment: comment_2,
-  image: image
-}
 
-requests = [request]
-library_service_client.discuss_book(requests).each do |element|
-  # Process element.
+require "library"
+
+# [START turing_prog_request_streaming_bidi]
+
+# Testing calling forms
+def sample_discuss_book(image_file_name)
+  # [START turing_prog_request_streaming_bidi_core]
+  # Instantiate a client
+  library_service_client = Library.new version: :v1
```

Comment:
For the client name variables, can we trim `service`?

```rb
library_client = Library.new version: :v1
```

Reply:
@jbolinger @landrito Is it OK to apply this change to incode samples as well? It doesn't seem to cause any problems but want to double check :) Plus if we can trim `service` everywhere it would make code in gapic-generator much cleaner,

Reply:
Sure, shorter sounds better to me too.